cluster/examples/kubernetes/ceph/object-openshift.yaml

```diff
@@ -95,3 +95,14 @@ spec:
     # Configure the pod liveness probe for the rgw daemon
     livenessProbe:
       disabled: false
+
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: rook-ceph-rgw-my-store
```

Comment:
Maybe it would be good to comment-doc this with something like
```yaml
# We recommend name to be the same as the service name below, but it is not required
```

Reply:
Done.